app.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var images = [];
+
+function Project (){
```

Comment:
There should be a parameter (like an object) so that you have something to populate the properties of the object you're making.